packages/history/index.ts

```diff
@@ -0,0 +1,437 @@
+////////////////////////////////////////////////////////////////////////////////
+//#region TYPES
+////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * Actions represent the type of change to a location value.
+ *
+ * @see https://github.com/remix-run/history/tree/main/docs/api-reference.md#action
```

Comment:
Can probably delete this since we aren't likely to publish docs just for the history lib.